requirements.txt

```diff
@@ -4,3 +4,4 @@ isodate
 pyparsing
 six
 SPARQLWrapper
+doctest-ignore-unicode
```

Comment:
@joernhees after I went to bed I realise that this should have been in `test_requires` in setup.py and not here :D 

And then we could run the tests with `python setup.py nosetests` on travis. 

But we can fix in master!